src/Logger/Adapter/Sentry.php

```diff
@@ -26,47 +26,19 @@ class Sentry extends Adapter
      * @var string (optional, the host where Sentry is reachable, in case of self-hosted Sentry could
      *              look like 'https://sentry.mycompany.com'. defaults to 'https://sentry.io')
      */
-    protected string $sentryHost;
+    protected string $sentryHost = 'https://sentry.io';
 
     /**
      * Sentry constructor.
      *
-     * @param  string  $dsn
+     * @param  string  $projectId
+     * @param  string  $key
+     * @param  string  $host
      */
-    public function __construct(string $dsn)
+    public function __construct(string $projectId, string $key, string $host)
```

Comment:
Same here, we can make host optional with default value

Reply:
done 👍